lib/autoload/filename_sanitizer.rb

```diff
@@ -0,0 +1,8 @@
+module FilenameSanitizer
```

Comment:
a\bc.txt is valid on Unix but invalid on Windows. I would need this for zip file uploads for the programming assessment templates, so it would be worth it.